docs/get-started/lotus/chain.md

```diff
@@ -17,113 +15,150 @@ Lotus will automatically sync to the latest _chain head_ by:
 - Fetching the block headers from the current _head_ down to the last synced epoch.
 - Retrieving and verifying all the blocks from the last synced epoch to the current head.
 
-Once Lotus is synced, it will learn about new blocks as they are mined for every epoch via Pubsub, and verify them accordingly. Note that in Lotus, every epoch might see a variable number of mined blocks.
+Once Lotus is synced, it will learn about new blocks as they are mined for every epoch and verify them accordingly. Every epoch might see a variable number of mined blocks.
 
-Filecoin's blockchain grows relatively fast, so a full sync will take long time. For this reason, Lotus offers a faster way to sync via trusted state snapshots.
+Filecoin's blockchain grows relatively fast, so a full sync will take a long time. For this reason, Lotus offers a faster way to sync using trusted state snapshots. There are two types of snapshot available:
 
-### Syncing from a trusted state snapshot (mainnet)
+| Name                                 | Description | End height    | Message start height        | State start height          |
+| ------------------------------------ | ----------- | ------------- | --------------------------- | --------------------------- |
+| [Lightweight](#lightweight-snapshot) | asd         | Current block | Current block - 2000 blocks | Current block - 2000 blocks |
+| [Full chain](#full-chain-snapshot)   | asd         | Current block | Genesis block               | Current block - 2000 blocks |
 
-We recommend most users to perform the initial node sync from a minimal, lightweight snapshot. Trusted state snapshots do not contain the full chain and might not be suitable for nodes that need to perform queries against historical state information, like block explorers, but otherwise work for most users.
+### Lightweight snapshot
 
-A recent minimal trusted state chain snapshot is available [here](https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/minimal_finality_stateroots_latest.car). We can instruct lotus to start the daemon and directly import from the URL:
+We recommend most users perform the initial node sync from a lightweight snapshot. These snapshots do not contain the full chain and are not suitable for nodes that need to perform queries against historical state information, such as block explorers. However, they are significantly smaller than full chain snapshots and should be sufficient for most use-cases.
 
-```sh
-# The snapshot size is about 7GiB. This works for mainnet.
-lotus daemon --import-snapshot https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/minimal_finality_stateroots_latest.car
+:::warning
+These lightweight state snapshots **do not contain any message receipts**. To get message receipts, you need to sync your Lotus node from the genesis block without using any of these snapshots.
```

Comment:
I believe, if you have synced the chain and have the states, then you can `set-head` to a previous epoch and resync from the peers to get EVERYTHING, including receipts. where the epoch one set-head to must be a block they have the state for. @ribasushi could you please confirm?

Reply:
@jennijuju not exactly. By default a `set-head` to something you do not already have in full locally will fail. You'd need to set the underdocumented `LOTUS_ENABLE_CHAINSTORE_FALLBACK=1`

Reply:
Just to make sure I understand you both correctly, in order for a user to get all the message receipts on their now after using a snapshot they need it:

- Enable `LOTUS_ENABLE_CHAINSTORE_FALLBACK=1`
- Use `set-head`.
- Wait for a very long time.

Is there any benefit to doing this, or would it be as fast/faster to just sync a node normally (without using a snapshot)?

Reply:
There is still a benefit - on each epoch transition you do not wait to get the next set of messages via libp2p. All your messages for "next tick" are already magically there: you just imported them as part of the 150+GiB (and growing ) snapshot.

Unfortunately "long time" and "slightly less long time" is pretty damn long either way ( ~13 days currently, given screaming hardware )

Reply:
Ok, that makes sense. I'll add these steps in.